kore/src/Prof.hs

```diff
@@ -62,6 +68,20 @@ instance (MonadMask prof, MonadProf prof) => MonadProf (ReaderT r prof) where
     traceEvent name = lift (traceEvent name)
     {-# INLINE traceEvent #-}
 
+instance
+    (MonadMask prof, MonadProf prof)
+    => MonadProf (Lazy.StateT s prof)
+  where
+    traceEvent name = lift (traceEvent name)
```

Comment:
Is `traceEvent` similar to `traceM`?

Reply:
`traceEvent` sends a message to the eventlog instead of to `stderr` like `traceM`.